.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+
+updates:
+  - package-ecosystem: "bundler"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-type: "all"
+    open-pull-requests-limit: 20
+    target-branch: "develop"
```

Comment:
Not sure that it's a good idea to specify target-branch, as I remember the previous version can automatically detect target-branch based on the repo configuration, so maybe we should remove it?